nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -254,6 +254,17 @@ language governing permissions and limitations under the License. -->
             <artifactId>org.everit.json.schema</artifactId>
             <version>1.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.fluenda</groupId>
+            <artifactId>ParCEFone</artifactId>
```

Comment:
L&N are areas we must be picky so I wil be happy to address the NOTICE. I promise one day I will get this thing right. :smiley: 

I will check if I can upgrade to 3.0.0 so to reduce the amount of "paperwork".